### PR TITLE
Style : Hover Effect & Color correction

### PR DESCRIPTION
**Pull request by Mouli-3542**
duplicate PR as #6

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -115,4 +115,22 @@ body {
     padding-top: 5px;
     padding-bottom: 5px;
     font-size: 11px;
+}
+.wrapper button {
```
Comment (malivinayak) [inline assets/style.css:120]: Add new line before `.wrapper button {`

File: assets/style.css [hunk 2]
```diff
@@ -115,4 +115,22 @@ body {
...
+    font-size: 18px;
+    padding: 14px 0;
+    border-radius: px;
+    background: #00539CFF;
+    transition: background 0.4s; /* Added a transition for smoother effect */
+}
+
+.wrapper button:hover {
+    background: #EEA47FFF; /* Darker orange color on hover */
 }
```
Comment (malivinayak) [inline assets/style.css:138]: Add new line at end

File: assets/style.css [hunk 3]
```diff
@@ -115,4 +115,22 @@ body {
...
+    margin-top: 20px;
+    font-size: 18px;
+    padding: 14px 0;
+    border-radius: px;
+    background: #00539CFF;
+    transition: background 0.4s; /* Added a transition for smoother effect */
+}
+
+.wrapper button:hover {
+    background: #EEA47FFF; /* Darker orange color on hover */
```
Comment (malivinayak) [inline assets/style.css]: update color to something calm as per your wish

File: assets/style.css [hunk 4]
```diff
@@ -115,4 +115,22 @@ body {
...
+    outline: none;
+    border: none;
+    cursor: pointer;
+    color: #fff;
+    margin-top: 20px;
+    font-size: 18px;
+    padding: 14px 0;
+    border-radius: px;
+    background: #00539CFF;
+    transition: background 0.4s; /* Added a transition for smoother effect */
```
Comment (malivinayak) [inline assets/style.css]: Remove the comment `/* Added a transition for smoother effect */`

**Review by malivinayak — CHANGES_REQUESTED** [review]
LGTM!

make requested changes I will merge the PR

File: assets/style.css (hunk 4 above)
Comment (Mouli-3542) [reply assets/style.css]: alright !

**Pull request closed without merging**